examples/serve/demo/cube-1-250-250.zarr/.zattrs

```diff
@@ -1,4 +1,4 @@
 {
     "Conventions": "CF-1.7",
-    "coordinates": "time_bnds lat_bnds lon_bnds"
+    "coordinates": "time_bnds lon_bnds lat_bnds"
```

Comment:
Strange, these should actually be listed in the order of their dimensions, which is 

```suggestion
    "coordinates": "time_bnds lat_bnds lon_bnds"
```

However, it doesn't seem to matter.